draft-dekater-scion-dataplane.md

```diff
@@ -204,7 +204,7 @@ SCION leverages source-based path selection, where path information is embedded
 
 **Path Transparency**: Path transparency is a property of a network architecture that gives endpoints full visibility over the network paths their packets are taking. Path transparency is weaker than path control.
 
-**SCMP**: SCION Control Message Protocol. SCMP is used for signaling connectivity problems, analogous to the Internet Control Message Protocol (ICMP). SCMP provides network diagnostic and error messages.
+**SCMP**: Future SCION Control Message Protocol. SCMP is intended to be analogous to the Internet Control Message Protocol (ICMP).
```

Comment:
```suggestion
```

Reply:
I can't figure what you're suggesting. The review UI seems to say that you want to remove the new text entirely as well as the old text. Is that what you're suggesting?